WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -127,15 +136,18 @@ private void StepOutputRegistrationPhase()
 					{
 						coinjoin.Inputs.Add(input);
 					}
+					round.LogInfo($"{coinjoin.Inputs.Count} inputs were added.");
 
 					// Add outputs:
 					foreach (var bob in round.Bobs)
 					{
 						coinjoin.Outputs.AddWithOptimize(bob.CalculateOutputAmount(round.FeeRate), bob.Script);
 					}
+					round.LogInfo($"{coinjoin.Outputs.Count} outputs were added.");
```

Comment:
This looks suspicious to me. The `foreach` loops through `round.Bobs` but you log `coinjoin.Outputs`.  

So either I would expect: 

`round.LogInfo($"{round.Bobs.Count} outputs were added.");` 

or

I would expect something like this: `round.LogInfo($"There is {coinjoin.Outputs.Count} outputs.");`

Reply:
Done.